andela/hc-leo
# [Feature #157694801] Job running too often

**What does this PR do?**
When a job is running too often i.e the job pings the check endpoint before the set period and outside the grace period, the user receives an alert informing them of this and the status of the of the job changes to often which is reflected on the users dashboard.

Screenshot of user's dashboard showing a job running too often:
<img width="1279" alt="screen shot v2" src="https://user-images.githubusercontent.com/25581395/41337822-ed33853a-6ef8-11e8-9a4f-88cebf5d4322.png">

Screenshot of email sent to the user:
<img width="886" alt="screen shot 2018-06-12 at 19 09 35" src="https://user-images.githubusercontent.com/25581395/41337857-13ef0d66-6ef9-11e8-94f0-2d50aea92dea.png">

**Description of task to be completed**
The user should be notified when a job is running too often as this is also an indication of a problem with the job

**How should this be manually tested?**
- Clone the repo, ```cd``` into the project and run ```./manage.py runserver```
- Navigate to the url indicated in your browser
- Create an account and/or login
- Create a new check and give it a name. The default grace and period times are suitable
- Ping the check
- Ping the check again
- You should receive a notification in your email telling you the job is running too often
- The status icon of the check should also change

**What are relevant pivotal tracker stories?**
[#157694801](https://www.pivotaltracker.com/story/show/157694801)
